.gitignore

```diff
@@ -3,3 +3,4 @@
 /.env
 /.vscode
 /.idea
+/.env.local
```

Comment:
/.env.local is not exists in this project

Reply:
My bad.